src/qt/forms/researcherwizardpoolpage.ui

```diff
@@ -91,7 +91,7 @@
    <item alignment="Qt::AlignVCenter">
     <widget class="QLabel" name="poolParagraphLabel">
      <property name="text">
-      <string>In pool mode, Gridcoin will not claim research rewards directly. The pool will collect and distribute research rewards on your behalf. Your wallet will earn staking rewards only (you do not need a BOINC account, CPID, or beacon). Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
+      <string>In this mode, the pool will take care of staking research rewards for you. You only need to sign up to the pool (BOINC account and beacon not needed). You will still be able to earn staking rewards regardless. Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
```

Comment:
I'm not sure about this one. We get tons of questions from new pool users who inadvertently set up the wallet for solo mode after reading some incomplete or outdated documentation floating around, so I wanted to directly address these points because the terms appear so frequently:

  - the wallet will not earn research rewards
  - the wallet can earn basic staking rewards
  - signing up for BOINC is not necessary
  - CPIDs are only relevant for solo mode
  - A beacon is only required for solo mode

Squashing all of that into natural language without sounding too techy is tricky. Maybe this is more clear:

```suggestion
      <string>In this mode, a pool will take care of staking research rewards for you. Your wallet can still earn standard staking rewards on your balance. You do not need a BOINC account, CPID, or beacon. Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
```

Suggestions still welcome.

Reply:
I am good with that.

Reply:
I agree on the new pool users point but doesn't the `do not need` part make it seem optional when it's not?

Also what's the reasoning behind `the pool` -> `a pool`?

Reply:
> I agree on the new pool users point but doesn't the do not need part make it seem optional when it's not?

Pool users can earn rewards without ever signing up for their own personal BOINC accounts. BOINC will crunch work under the pool's CPIDs. It's possible for someone to set up dual pool/solo mining, but most users will pick one or the other.

> Also what's the reasoning behind the pool -> a pool?

Just wanted to change the style to decrease the chance of a user assuming that there is a single official pool.